nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -2168,4 +2190,46 @@ public void varArgsWithTypeUseAnnotationMethodSerializationTest() {
         .setOutputFileNameAndHeader(ERROR_FILE_NAME, ERROR_FILE_HEADER)
         .doTest();
   }
+
+  @Test
+  public void anonymousSubClassConstructorSerializationTest() {
+    SerializationTestHelper<ErrorDisplay> tester = new SerializationTestHelper<>(root);
+    tester
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
+        .addSourceLines(
+            // toString() call on method symbol will serialize the annotation below which should not
+            // be present in string representation fo a method signature.
```

Comment:
Which annotation? (Test looks fine to me, but comment seems unrelated. Also, this test case is subtle enough that rather than just removing the comment, if it's an incorrect copy paste, I'd rewrite it to explain what we are actually testing: correct serialization of names for constructors invoked while creating anonymous inner classes, where the name is technically the name of the super-class, not of the anonymous inner class)

Reply:
Yes sorry mistake, fixed it in b7679073e2623fba5e5b7cef628ba9c590f409d7 and added your explanation.